include/grpc++/time.h

```diff
@@ -85,6 +85,9 @@ namespace grpc {
 // from and to should be absolute time.
 void Timepoint2Timespec(const std::chrono::system_clock::time_point& from,
                         gpr_timespec* to);
+void TimepointHR2Timespec(
+    const std::chrono::high_resolution_clock::time_point& from,
+    gpr_timespec* to);
```

Comment:
the formatting of these three lines looks funny. Is this how clang-format does it?